test/acceptance/features/unreadyServiceBinding.feature

```diff
@@ -0,0 +1,116 @@
+Feature: Only Unready Service Binding Resources Can Be Changed
+
+    As a user of Service Binding operator
+    I should be able to modify a Service Binding if it is not yet ready
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+        * CustomResourceDefinition backends.stable.example.com is available
+
+    Scenario: Allow modifying a Service Binding when it is not yet ready due to "Application Not Found"
```

Comment:
Scenario should be the opposite of scenario present in the the ready service binding feature file.

This scenario should be moved next to the one in ready service binding feature file.